src/backend/common/Logger.hpp

```diff
@@ -11,23 +11,7 @@
 
 #include <memory>
 #include <string>
-
-#ifdef AF_WITH_LOGGING
 #include <spdlog/spdlog.h>
```

Comment:
You can forward declare spdlog::logger instead of including the entire header.

Reply:
Good catch - addressed.